www/js/lib/uiUtil.js

```diff
@@ -32,6 +32,48 @@ if (webpMachine) {
 }
 
 define(rqDef, function(settingsStore) {
+
+    /**
+     * Displays a Bootstrap alert or confirm dialog box depending on the options provided
+     * 
+     * @param {String} message The alert message to display in the body of the modal 
+     * @param {String} label The modal's label or title which appears in the header (optional, Default = "Confirmation" or "Message")
+     * @param {Boolean} isConfirm If true, the modal will be a confirm dialog box, otherwise it will be a simple alert message 
+     * @param {String} declineConfirmLabel The text to display on the decline confirmation button (optional, Default = "Cancel") 
+     * @param {String} approveConfirmLabel  The text to display on the approve confirmation button (optional, Default = "Confirm")
+     * @param {String} closeMessageLabel  The text to display on the close alert message button (optional, Default = "Okay")
+     * @returns {Promise<Boolean>} A promise which resolves to true if the user clicked Confirm, false if the user clicked Cancel/Okay, backdrop or the cross(x) button
+     */
+    function systemAlert(message, label, isConfirm, declineConfirmLabel, approveConfirmLabel, closeMessageLabel) {
+        declineConfirmLabel = declineConfirmLabel || "Cancel";
+        approveConfirmLabel = approveConfirmLabel || "Confirm";
+        closeMessageLabel = closeMessageLabel || "Okay";
+        label = label || (isConfirm ? "Confirmation" : "Message");
+        return new Promise(function (resolve, reject) {
+            if (!message) reject("Missing body message");
+            // Set the text to the modal and it's buttons
+            document.getElementById("approveConfirm").textContent = approveConfirmLabel;
+            document.getElementById("declineConfirm").textContent = declineConfirmLabel;
+            document.getElementById("closeMessage").textContent = closeMessageLabel;
+            document.getElementById("modalLabel").textContent = label;
+            document.getElementById("modalText").textContent = message;
+            // Display buttons acc to the type of alert
+            document.getElementById("approveConfirm").style.display = isConfirm ? "block" : "none";
```

Comment:
@gaurav7019 I found a small issue while backporting this code downstream to KJSW. Due to using an older version of Bootstrap (maybe) downstream, instead of "block", you should use "inline" here and in the next two lines. It's needed for KJSW, or else I get the anomaly in screenshot below. NB I don't get this in KJS, so it's OK with Bootstrap 4, but I tested "inline" in KJS and it works fine.

![image](https://user-images.githubusercontent.com/4304337/152636748-7a627cdf-ac5c-4fdb-baa2-d98c0cf90434.png)

Reply:
Yes, inline would be better. Will do.